src/main/java/uk/gov/companieshouse/api/accounts/model/filing/Data.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.api.accounts.model.filing;
+
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Data {
+
+    @JsonProperty("links")
+    private List<Link> links;
+
+    @JsonProperty("period_end_on")
+    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd")
```

Comment:
I'm not sure that we need this anymore, the way Abridged handled dates as the old way. We should be using `java.time.LocalDate` from here on out and it actually defaults to the correct format that we want.

Reply:
copy and paste from abridged. I will change it to use localdate`